env-var-docs/parser-package/src/env_var_docs/templates/SettingsManifest.java.jinja

```diff
@@ -43,14 +43,15 @@ public final class SettingsManifest extends AbstractSettingsManifest {
   }
 
   {% for method_spec in getter_method_specs %}
+  {% if method_spec.variable.mode == writeable_mode %}
   /** {{method_spec.doc()}} */
-  public {{method_spec.return_type()}} get{{method_spec.method_name()}}() {
-    return {{method_spec.internal_getter()}}("{{method_spec.variable_name()}}");
+  public {{method_spec.return_type()}} get{{method_spec.method_name()}}(RequestHeader request) {
+    return {{method_spec.internal_getter()}}("{{method_spec.variable_name()}}", request);
   }
-  {% if method_spec.variable.type == "bool" %}
+  {% else %}
   /** {{method_spec.doc()}} */
-  public {{method_spec.return_type()}} get{{method_spec.method_name()}}(Request request) {
-    return {{method_spec.internal_getter()}}("{{method_spec.variable_name()}}", request);
+  public {{method_spec.return_type()}} get{{method_spec.method_name()}}() {
+    return {{method_spec.internal_getter()}}("{{method_spec.variable_name()}}");
   }
```

Comment:
💁🏻‍♂️  there are two varieties of getter method for a setting: one that takes a request object and one that does not. The former is for writeable settings since the admin-specified values are stored on the request object for reference, while the latter is for settings that are only read from Config.